locktree/tests/lock_request_start_retry_race.cc

```diff
@@ -97,20 +89,13 @@ static void run_locker(locktree *lt, TXNID txnid, const DBT *key) {
 
         toku_pthread_yield();
         if ((i % 10) == 0)
-            std::cout << toku_pthread_self() << " " << i << std::endl;
+            std::cout << std::this_thread::get_id() << " " << i << std::endl;
```

Comment:
Why not to use std::iostream?  std::this_thread::get_id() returns std::thread::id class. This is not uint64_t. This class has an implementation of '<<' operator which serializes thread id. So if we are going to use fprintf() we have also to use something like std::stringstream to serialize thread id. If so then why not to use std::cerr?